demos/shadow/shadow_demo_main/demo_config.h

```diff
@@ -117,40 +117,46 @@
  *
  * No two clients may use the same client identifier simultaneously.
  */
-#define CLIENT_IDENTIFIER      "testclient"
+#define CLIENT_IDENTIFIER         "testclient"
 
 /**
  * @brief Size of the network buffer for MQTT packets.
  */
-#define NETWORK_BUFFER_SIZE    ( 1024U )
+#define NETWORK_BUFFER_SIZE       ( 1024U )
 
 /**
  * @brief The name of the operating system or SDK that the application is
  * running a part of.
  */
-#define SDK_NAME                  "aws-iot-device-sdk-embedded-C"
+#define OS_NAME                   "Ubuntu"
```

Comment:
I think it will be better to mention this as example values in comments. What do you think?

Reply:
Yes I will mention that Ubuntu is an example value and that it should be updated to the current user's OS.